add pages for China KCS 2023: kubernetes contributor summit (#417)
kubernetes/contributor-site: content/en/events/2023/kcscn/_index.md

---
title: "Contributor Summit China 2023"
type: docs
weight: 1
description: >
  Kubernetes Contributor Summit China, hosted in Shanghai.

---

### About

The China Summit will be a lightweight event taking place on September 26th

> in Shanghai, The China alongside 

Reviewer: ```suggestion
in Shanghai, China alongside 
```

<a href="https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/" rel="noopener noreferrer" target="_blank">KubeCon + CloudNativeCon + Open Source Summit</a>
at the
<a href="https://en.shcec.com.cn/" rel="noopener noreferrer" target="_blank">Shanghai Convention & Exhibition Center Of International Sourcing</a>.

The Summit will be a loosely organized half-day event with a [relaxed schedule](/events/2023/kcseu/schedule),
 including three 30-minutes talks, 1.5 hours of Unconference Sessions, and 
potential SIG and WG group meetings.

This page will be updated as more activities are scheduled. If you have
questions or comments, please reach out to the Summit Staff by emailing
community@kubernetes.io or asking in the
<a href="https://kubernetes.slack.com/messages/contributor-summit" rel="noopener noreferrer" target="_blank">#contributor-summit</a>
slack channel.

[location]: /events/2023/kcscn/location/

### Recordings

> This event is in plan. Find materials here later:

Reviewer: ```suggestion
This event is in planning. Find materials here later:
```

- recordings
- photos

### Call for Papers

> A [CFP] for sessions will be open through Monday, August 14th.

Reviewer: ```suggestion
A [CFP] for sessions will be open from Monday, August 14th.
```

> [CFP]:  TODO-Add-link

Reviewer 2: Updated.

### Code of Conduct

Attendees agree to abide by Kubernetes [Code of Conduct]. To report an incident
please reach out to the [CNCF event staff].

[Code of Conduct]: /community/code-of-conduct

> [CNCF event staff]: https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/contact-us/

Reviewer 3: ```suggestion
[CNCF event staff]: https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/attend/code-of-conduct/#if-you-witness-unacceptable-behavior-cn
```
that has direct contacts for CoC issues 👍

> [email us]: mailto:community@kubernetes.io or mailto:community@cncf.io

Reviewer 3: ```suggestion
```
the reference link wouldn't work as is, but between the earlier email link and the CoC links, should be good to remove.
